src/libraries/System.Net.Sockets/tests/FunctionalTests/SendReceive/SendReceiveCommon.cs

```diff
@@ -0,0 +1,286 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
```

Comment:
"Common" is kind of a strange term for this... it seems like it's really just a handful of send/receive related tests that don't fit elsewhere. I'd suggest calling it "SendReceiveMisc.cs" or something like that.